code/game/objects/items/weaponry.dm

```diff
@@ -1083,3 +1083,15 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 		balloon_alert(user, "you're too weak!")
 		return
 	return ..()
+
+/obj/item/melee/error
```

Comment:
Being able to intentionally buy a bugged object might be a little too meta.

Reply:
I think trying to make it less meta might spoil the joke, so probably better off just axing it if its too much.